OSDOCS-11735: Update GCP installation info (#81053)
openshift/openshift-docs: modules/minimum-required-permissions-ipi-gcp-xpn.adoc

* `roles/compute.securityAdmin`
====

> .Required permissions for creating service account resources

Reviewer: In fact, these permissions are for getting/setting the project's IAM policy in the host project, e.g. granting the service account(s) the required permissions in the host project.

> .Required permissions for granting the service account the required permissions in the host project

Reviewer: The line 18 tells "_Ensure that the host project applies **one of the following configurations** to the service account:_", which lists 3 following sections, 
- Required permissions for creating firewalls in the host project
- Required permissions for granting the service account the required permissions in the host project
- Required minimal permissions

In fact, the 2nd section applies, for both the 1st and the 3rd. 

How about re-org like below? 
------
Ensure that the host project applies one of the following configurations to the service account:

- Required permissions for creating firewalls in the host project
- Required permissions for without creating firewalls in the host project

Besides, if you do not supply a service account for control plane nodes in the `install-config.yaml` file, please also grant below permissions to the service account in the host project: 
* `resourcemanager.projects.getIamPolicy`
* `resourcemanager.projects.setIamPolicy`

Reviewer: @barbacbd Had you got a chance reviewing this comment and the last comment? Thanks!

[%collapsible]
====
* `resourcemanager.projects.getIamPolicy`
* `resourcemanager.projects.setIamPolicy`
====

[IMPORTANT]
====

> These permissions are required when the user does not supply a service account for compute and control plane nodes in the install config. The permissions should be granted in the host project.

Reviewer 2: ```suggestion
The following permissions are required if you do not supply a service account for compute and control plane nodes in the `install-config.yaml` file. The permissions must be granted in the host project.
```

I would recommend changing this to "the following permissions" and move the note to right before line 28, so customer reads the note before reading the permissions.

> These permissions are required when the user does not supply a service account for compute and control plane nodes in the install config. The service account(s) should exist in the host project.

Reviewer 3: > The service account(s) should exist in the host project.

Let's rephrase this a bit. The service account can be from any project, but the permissions need to be granted for the host project. So it might be more clear to say something like "these permissions should be granted in the host project."

> The following permissions are required if you do not supply a service account for compute and control plane nodes in the `install-config.yaml` file. The permissions must be granted in the host project.

Reviewer: - Do you mean "The above permissions are..." instead? 
- The permissions are required only if you do not supply a service account for **control plane** nodes in the `install-config.yaml` file. Please refer to [my testing](https://issues.redhat.com/browse/OSDOCS-11735?focusedId=25568016&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25568016), thanks!
